site/content/en/docs/drivers/qemu.md

```diff
@@ -13,6 +13,14 @@ The `qemu` driver users QEMU (system) for VM creation.
 
 <https://www.qemu.org/>
 
+## Special features
+
+minikube start supports some qemu specific flags:
+
+* **`--qemu-firmware-path`**: The path to the firmware image to be used.
+  * Note: if this flag does not take effect, try removing the file `~/.minikube` which may have a reference to this setting.
```

Comment:
this hopefully should not happen as the flag should override config
but still we should never ask users to delete the config manually, this will make any existing clusters stall and abandonded
instead "minikube delete --all --purge"

Reply:
Very much appreciated, have updated the documentation as suggested.

Glad to learn that simply removing the `~/.minikube` home directory will cause the existing clusters to stall and become abandoned. In that case I think that those processes would need to be stopped manually or could be stopped by a machine restart.